File: 백준/Silver/2563. 색종이/색종이.py
import sys
input = sys.stdin.readline
    
T = int(input())

result = 0
paper = [[0] * 100 for _ in range(100)]

for i in range(T):
    blackPaper = list(map(int, input().split()))

    for j in range(blackPaper[0], blackPaper[0] + 10) :
        for k in range(blackPaper[1], blackPaper[1] + 10) :
            paper[j][k] = 1

for i in range(100) :
    result += sum(paper[i])
print(result)

File: 백준/Silver/2563. 색종이/색종이.java
import java.io.*;
import java.util.*;

public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		StringBuilder sb = new StringBuilder();
		StringTokenizer st = new StringTokenizer(br.readLine());

		int N = Integer.parseInt(st.nextToken());

		boolean[][] v = new boolean[101][101];
		int result = 0;
		
		for (int paper = 0; paper < N; paper++) {
			st = new StringTokenizer(br.readLine());
			int x = Integer.parseInt(st.nextToken());
			int y = Integer.parseInt(st.nextToken());
			
			for (int i = x; i < x+10; i++) {
				for (int j = y; j < y + 10; j++) {
					v[i][j] = true;
				}
			}
		}
		
		for (int i = 0; i <= 100; i++) {
			for (int j = 0; j <= 100; j++) {
				if (v[i][j]) result++;
			}
		}
		
		System.out.println(result);
		br.close();
	}
}